.env

```
@@ -0,0 +1,10 @@
+# Postgres Docker
+POSTGRES_PORT=5432
+POSTGRES_DB=goggles
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=admin12
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_PORT key

(UnorderedKey)